scripts/utilities/update_revision.sh

```diff
@@ -51,6 +51,12 @@
 #  commit but reuses the Change-Id. i.e. it updates the pending change with a
 #  new patch set.
 
+# IMPORTANT: if you change this script in any way, please update the
+# LAST_AUTHOR variable, using the format "your name <your email>".
+# This is used for the required Signed-off-by line in commits when
+# using GitHub.
+LAST_AUTHOR="Denilson Antonio Marques Junior <denimj@linux.vnet.ibm.com>"
```

Comment:
This change doesn't make sense IMO, in this way the person who wants to use the scripts would need to change this file.
The patch needs to be signed off by the person who wrote it.